tests/assert/assert.go

```diff
@@ -21,3 +21,13 @@ func Error(t *testing.T, err error, expected string) {
 		t.Fatalf("expected '%s' but got '%s'", expected, err.Error())
 	}
 }
+
+// True asserts that an interface is equal to boolean "true"
+func True(t *testing.T, expected interface{}) {
```

Comment:
I didn't include it yet just because we only had the two assert methods we needed. I'm open to including it whenever we feel like we've crossed the line where it's silly to implement ourselves. So if you feel like it makes sense now then feel free to go for it.